page_code.html

```diff
@@ -0,0 +1,504 @@
+<!DOCTYPE html>
```

Comment:
we do not need html files to be saved as the goal of this lab is to collect a dataset that consists of texts

Reply:
are you planning on deleting it? :)